app/index.html

```diff
@@ -247,8 +247,10 @@ <h1>About Myself</h1>
       </div>
       <div class="card-description about-body">
         <p>
```

Comment:
1. Please The page should be  divided half of it contains the project's picture and the other half contains the project's description, these switch places with each project (picture is on the left and description on the right for the first project, description on the left and picture on the right for the second project and so on and so forth).
1. The contents of the description are aligned to the left.
your code 
![image](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/74744787/a2a2debe-252e-4a97-97bf-8efac2687f41)
Figma 
![image](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/74744787/d7985012-79e3-4223-a5a2-c0e3aaecf092)

1. Please below the heading there should be a row with info about the work (if you don't have previous programming experience it would be Microverse for all of them) the position (Full Stack Dev) and the year the project was build, separated by grey circles.
your code 
![image](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/74744787/d778dfd8-e42e-4dee-890c-901217abdffe)
Figma 
![image](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/74744787/6db02936-4015-41e4-892e-33e9b76cfae5)

1. Please The project description text should be smaller than the row with info and it should take 3 to 4 lines of text.
![image](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/74744787/2c4342da-6d47-4a1a-a8d1-702cbd190afe)

1. After the project description there should be some button like tags listing all the technologies that were used to build the project.
![image](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/74744787/77e8eba5-92a4-454f-b61f-b9a094c9a457)
- Please fix the styling of the see project button it should match the design 
![image](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/74744787/7af26ed9-1638-4cf5-b8cd-fda8d8109cd0)
![image](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/74744787/a9bac171-a34a-44c6-9b30-94a9c2bd8a0c)